mache-chroniclemap/src/main/java/com/excelian/mache/chroniclemap/solr/ConcurrentLRUCache.java

```diff
@@ -268,20 +263,18 @@ private void markAndSweep() {
                         newOldestEntry = Math.min(thisEntry, newOldestEntry);
                     }
                 }
-                // System.out.println("items removed:" + numRemoved + " numKept=" + numKept + " esetSz="+ eSize + " sz-numRemoved=" + (sz-numRemoved));
             }
 
 
             // if we still didn't remove enough entries, then make another pass while
             // inserting into a priority queue
-            if (sz - numRemoved > acceptableWaterMark) {
+            if (currentSize - numRemoved > acceptableWaterMark) {
```

Comment:
yeah i didn't want to mess with this code too much as it's the solr stuff, was just trying to work through it myself and wanted a better name than sz